WWDC/LiveObserver.swift

```diff
@@ -70,27 +70,35 @@ final class LiveObserver {
     }
 
     func refresh() {
-        specialEventsObserver.fetch()
+        NSObject.cancelPreviousPerformRequests(withTarget: self)
```

Comment:
This is a good change, using delorean I could start live sessions then they would be stuck because the live observer never started to clear the live flag. Now this should give a way to override it.